tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -135,6 +135,10 @@ public function testListTableWithBinary()
 
     public function testNonDefaultPKOrder()
     {
+        if (!class_exists('SQLite3')) {
```

Comment:
It should be `extension_loaded('sqlite3')` and should also be part of another PR (as stated above).